test/compile/selection/inputs.test.ts

```diff
@@ -154,4 +165,32 @@ describe('Inputs Selection Transform', () => {
       ])
     );
   });
+
+  it('respects initialization', () => {
+    model.component.selection = {seven: selCmpts['seven']};
+    expect(selection.assembleUnitSelectionSignals(model, [])).toEqual(
+      expect.arrayContaining([
+        {
+          name: 'seven_tuple',
+          update: 'seven_Year !== null ? {fields: seven_tuple_fields, values: [seven_Year]} : null'
+        }
+      ])
```

Comment:
is only one entry here. Why use arrayContaining?

Reply:
This is not created in this PR, so feel free to send a follow up fix.